gradle.properties

```diff
@@ -16,3 +16,4 @@ org.gradle.jvmargs=-Xmx4096m
 # org.gradle.parallel=true
 android.useAndroidX=true
 android.enableJetifier=true
+android.nonTransitiveRClass=false
```

Comment:
Another AGP breaking behavioral change – it doesn't generate transitive R classes by default anymore to squeeze the performance. We need to have to in order to share icon ids from the common module